### PR TITLE
Remove incorrect HTML self close tag

### PR DESCRIPTION
**Pull request by wxiaoguang**
HTML is not XML.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by KN4CK3R** [comment]
🚀

File: templates/user/profile.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		<div class="ui stackable grid">
 			<div class="ui five wide column">
 				<div class="ui card">
-					<div id="profile-avatar" class="content gt-df"/>
+					<div id="profile-avatar" class="content gt-df">
```
Comment (delvh) [inline templates/user/profile.tmpl:7]: Did the logic change here?
Looks a bit like it to me…

File: templates/repo/graph/svgcontainer.tmpl [hunk 2]
```diff
@@ -14,9 +14,9 @@
 					{{- else if eq $glyph.Glyph '_' -}}
 						M {{Add (Mul $glyph.Column 5) 0}} {{Add (Mul $glyph.Row 12) 12}} h 10 {{/* */ -}}
 					{{- end -}}
-				{{- end}}" stroke-width="1" fill="none" id="flow-{{$flow.ID}}-path" stroke-linecap="round"/>
+				{{- end}}" stroke-width="1" fill="none" id="flow-{{$flow.ID}}-path" stroke-linecap="round">
 				{{range $flow.Commits}}
-					<circle class="flow-commit" cx="{{Add (Mul .Column 5) 5}}" cy="{{Add (Mul .Row 12) 6}}" r="2.5" stroke="none" id="flow-commit-{{.Rev}}" data-rev="{{.Rev}}"/>
+					<circle class="flow-commit" cx="{{Add (Mul .Column 5) 5}}" cy="{{Add (Mul .Row 12) 6}}" r="2.5" stroke="none" id="flow-commit-{{.Rev}}" data-rev="{{.Rev}}">
```
Comment (delvh) [inline templates/repo/graph/svgcontainer.tmpl:19]: https://stackoverflow.com/a/24301479

**Review by delvh — COMMENTED** [review]
Seems like your regex had three false positives.

File: templates/user/profile.tmpl (hunk 1 above)
Comment (wxiaoguang) [reply templates/user/profile.tmpl:7]: It is right, I tested it.

Comment (KN4CK3R) [reply templates/user/profile.tmpl:7]: The real closing `</div>` is in line 17.

File: templates/repo/graph/svgcontainer.tmpl (hunk 2 above)
Comment (wxiaoguang) [reply templates/repo/graph/svgcontainer.tmpl:19]: Hmm, SVG is different, I will check it.

File: templates/user/profile.tmpl (hunk 1 above)
Comment (delvh) [reply templates/user/profile.tmpl:7]: And how has this worked previously?
Or has it always been wrong before?

Comment (wxiaoguang) [reply templates/user/profile.tmpl:7]: No, `<div />` doesn't close.

`<div />....</div>` works like `<div>....</div>`

File: templates/repo/graph/svgcontainer.tmpl (hunk 2 above)
Comment (wxiaoguang) [reply templates/repo/graph/svgcontainer.tmpl:19]: Fix SVG close tag #23751

**Pull request merged**